docs/intro/install/ubuntu.rst

```diff
@@ -7,7 +7,7 @@ Installing Elgg on Ubuntu Linux
 
      sudo apt-get install apache2
      sudo apt-get install mysql-server
-     sudo apt-get install php5 libapache2-mod-php5 php5-mysql
+     sudo apt-get install php5 libapache2-mod-php5 php5-mysqlnd
```

Comment:
I just figured it's better http://serverfault.com/questions/400434/php-mysqlnd-or-php-mysql